src/index.js

```diff
@@ -85,6 +93,12 @@ class Game extends React.Component {
     });
   }
 
+  sortHistory() {
+    this.setState({
+      isDescending: !this.state.isDescending,
```

Comment:
I'm still learning React myself so I don't pretend to know about best practices and whatnot, but I feel like the code would be a bit more readable if instead of hiding the initialization of this state variable halfway down the page inside the `sortHistory()` method, we instead initialize it with a value of `false` in the constructor of the `Game` class. This helps the reader more quickly be aware of all of the values that will be used for state.

Reply:
...also I think you have the word wrong here. By default the code lists the most recent move first, then "descends" down the the starting (zeroth) move. So by default the list is already in descending order, and when we first call `sortHistory` the opposite of an undefined value (i.e. `true`) is assigned to `isDescending`, but actually at that point the list would be sorted in ascending order after a single click.

Reply:
> I'm still learning React myself so I don't pretend to know about best practices and whatnot, but I feel like the code would be a bit more readable if instead of hiding the initialization of this state variable halfway down the page inside the `sortHistory()` method, we instead initialize it with a value of `false` in the constructor of the `Game` class. This helps the reader more quickly be aware of all of the values that will be used for state.

I like that a lot. I agree!

Reply:
> ...also I think you have the word wrong here. ~By default~ Without clicking anything, the code lists the most recent move first and then "descends" down to the starting (zeroth) move. So by default the list is already in descending order, and when we first call `sortHistory` the ~opposite~ boolean negation of an undefined value (i.e. `true`) is assigned to `isDescending`, but actually at that point the list would be sorted in ascending order after that single click.

Changing the word sounds good to me also.
It should read 'isAscending' instead if I understand correctly

Reply:
> > I'm still learning React myself so I don't pretend to know about best practices and whatnot, but I feel like the code would be a bit more readable if instead of hiding the initialization of this state variable halfway down the page inside the `sortHistory()` method, we instead initialize it with a value of `false` in the constructor of the `Game` class. This helps the reader more quickly be aware of all of the values that will be used for state.
> 
> I like that a lot. I agree!

Seems like folks like [separating it into a method ](https://stackoverflow.com/questions/40359800/how-to-toggle-boolean-state-of-a-react-component) and I couldn't get it to work the way you suggested :(

Reply:
Changed the variable name to isAscending